src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+import { TodoServiceErrors } from '../types/TodoServiceErrors';
+
+interface PropsErrorNotification {
+  message: TodoServiceErrors | null;
+  onClose: () => void;
+}
+
+export const ErrorNotification: React.FC<PropsErrorNotification> = ({
+  message,
+  onClose,
+}) => {
+  useEffect(() => {
+    if (message) {
+      const timer = setTimeout(() => {
+        onClose();
+      }, 3000);
+
+      return () => clearTimeout(timer);
+    }
+
+    return undefined;
+  }, [message, onClose]);
```

Comment:
```suggestion
  useEffect(() => {
    if (message) {
      const timer = setTimeout(() => {
        onClose();
      }, 3000);

      return () => clearTimeout(timer);
    }

    return undefined;
  }, [message, onClose]);
```
what if we start if with if(!message) ?)